libs/vertexai/langchain_google_vertexai/functions_utils.py

```diff
@@ -54,7 +54,7 @@ def _format_tool_to_vertex_function(tool: BaseTool) -> FunctionDescription:
                     }
                     for k, v in schema["properties"].items()
                 },
-                "required": schema["required"],
+                "required": schema.get("required", []),
```

Comment:
nits: should we maybe just remove `requried` arg if it's empty? it looks like it's not a required one